public/app/frontend/src/components/navigation-main/navigation-main.html.twig

```diff
@@ -26,7 +26,7 @@ example:
         ]
     }%}
 #}
-<nav id="navigation-main" class="navigation-main">
+<nav id="navigation-main" class="navigation-main" aria-label="Primary">
```

Comment:
Does the main navigation need Primary as an aria label, and will the end user that requires this label understand the context?

Reply:
@wilson1000 Good question. In this case we need an aria label because the justice site has multiple nav bars, this one and the sidebar. Whether Primary and Secondary are enough to explain their purpose I'm not sure, but it does at least differentiate between them. 

The MOJ design system example (https://design-patterns.service.justice.gov.uk/components/primary-navigation/) uses primary.

I think ideally we would have a "mega menu" as the main menu which integrates the dynamic parts of the sidebar, but as we can't change the page layout we're a bit limited unfortunately.